test/check-machines-create

```diff
@@ -375,6 +375,15 @@ class TestMachinesCreate(VirtualMachinesCase):
         self.machine.execute("virsh undefine pxe-guest")
         runner.checkEnvIsEmpty()
 
+        # Check postponed starting
```

Comment:
There is a whole test for this already, testCreateThenInstall.

Maybe we want to just extend that one for the state check?

Reply:
I'm happy to move it there, sure! I'll still let the tests finish here, to check if the changed option breaks anything else.

Reply:
Done. I validated that this still reproduces the original bug.